cves/kernel/CVE-2021-45402.yml

```diff
@@ -151,10 +158,15 @@ unit_tested:
 
```

Comment:
Upvotes: 5. Very well detailed, good work!

Reply:
2 upvotes, good job!

Reply:
Thank you!